README.md

```diff
@@ -684,22 +685,45 @@ Special characters can be included:
 * Single Quote: `\'`
 * Unicode scalar: `\u{n}` where n is between one and eight hexadecimal digits
 
-#### Array Access Syntax
+The built-in methods and accessors for Strings are different in Swift 1.x, 2.1+, and 3.0, and so are not covered here.
```

Comment:
You can likely leave this out since the intention of this cheat sheet is to be for Swift 3+ (and eventually will be Swift 4+ once that's released)